worker/dbaccessor/worker.go

```diff
@@ -416,10 +417,22 @@ func (w *dbWorker) initialiseDqlite(options ...app.Option) error {
 		return errors.Trace(err)
 	}
 
-	ctx, cancel := w.scopedContext()
-	defer cancel()
+	ctx, pCancel := w.scopedContext()
+	defer pCancel()
+	ctx, cCancel := context.WithTimeout(ctx, time.Minute)
+	defer cCancel()
```

Comment:
It would be better if we could combine the contexts into one context see: https://github.com/spoke-d/task/blob/master/conjoint/context.go

Reply:
I actually _'d out the second one, because the parent will effectively cancel the child, but the linter complained.
Don't really want to bring in a new dependency for the one case.